docs/setting-up.md

```diff
@@ -18,11 +18,11 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
 1. Install Python 2.7.
 1. Install Node.js (minimum version 6.11.5).
 
-## Step 2: Obtain your own repository copy
+## Step 2: Obtain your own copy of the repository
 
-1. Fork our repo at https://github.com/TEAMMATES/teammates. Clone that fork to your hard disk.
+1. Fork our repo at https://github.com/TEAMMATES/teammates. Clone the fork to your hard disk.
 
-1. Add a remote name (e.g `upstream`) for the main repo for your repo to keep in sync with, and then fetch the remote-tracking branches from the main repo.
+1. Add a remote name (e.g upstream) for your copy of the main repo. Fetch the remote-tracking branches from the main repo to keep it in sync with your copy.
```

Comment:
Let's keep the ***`*** around upstream as its a special word in this context

Reply:
I missed the single quotes. Will change it.